main/unwindi.c

```diff
@@ -262,7 +271,12 @@ extern MIOPos uwiGetFilePosition (void)
 
 extern void uwiPushMarker (void)
 {
-	Assert (uwiCurrentMarker - uwiMarkerStack < 255);
+	if (uwiCurrentMarker - uwiMarkerStack >= ( uwiMarkerStackLength - 1 )) {
+		error (WARNING,
+			"unwindi.c: trying to add too many markers "
+			"(this is a bug, please consider filing an issue)");
+		uwiCurrentMarker = NULL;
+	}
```

Comment:
```
		error (WARNING,
			"unwindi.c: trying to add too many markers during  parsing: %s"
			"(this is a bug, please consider filing an issue)", getInputFileName());
```

Reply:
My comment was wrong. The source file of ctags is not needed. Instead, the INPUT source file must be printed.

Reply:
OK, I'll fix it.